src/Microsoft.Health.Fhir.Core/Features/Conformance/Serialization/EnumLiteralJsonConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Reflection;
+using EnsureThat;
+using Hl7.Fhir.Utility;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Conformance.Serialization
+{
+    internal class EnumLiteralJsonConverter : JsonConverter
+    {
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            EnsureArg.IsNotNull(writer, nameof(writer));
+            EnsureArg.IsNotNull(serializer, nameof(serializer));
+
+            if (value is Enum obj)
```

Comment:
FhirUtility already has a method called `GetLiteral`, which I think does this. It's in Hl7.Fhir.Support NuGet package which i think is version agnostic?